social/event/v0.1.0/schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/social/event/v0.1.0/schema.json",
+    "title": "Event",
+    "description": "Schema for a calendar event",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4285F4",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "social_event",
+        "indexes": {
+            "name": ["name"],
+            "startTime": ["start.dateTime", "start.date"],
+            "endTime": ["end.dateTime", "end.date"],
+            "source": ["sourceApplication", "sourceId"]
+        }
+    },
+    "layouts": {
+        "create": ["name", "start", "end", "location"],
+        "view": ["name", "description", "start", "end", "location", "organizer", "attendees", "status"]
```

Comment:
Must specify specific fields (ie: `start.date`)

Reply:
> Must specify specific fields (ie: `start.date`)

Could add more quotes here @tahpot ?
Tbh, I am not sure what to add in `layouts.create` and `layout.view` fileds.